pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -148,6 +148,12 @@ type AlertmanagerSpec struct {
 	// and the actual ExternalURL is still true, but the server serves requests
 	// under a different route prefix. For example for use with `kubectl proxy`.
 	RoutePrefix string `json:"routePrefix,omitempty"`
+	// Maximum number of GET requests processed concurrently.
+	// +optional
+	GetConcurrency *uint32 `json:"getConcurrency,omitempty"`
+	// Timeout for HTTP requests.
+	// +optional
+	RequestTimeout *uint32 `json:"requestTimeout,omitempty"`
```

Comment:
Please feel free to suggest better names. I was also thinking if it makes sense to move web.* to a WebConfig struct but the RoutePrefix is already in am.Spec.

Reply:
We could move the new fields to `AlertmanagerWebSpec`.